courses/models.py

```
@@ -38,25 +48,40 @@
         return self.teachers.all()
 
 
-class ClassroomTeacher(BaseModel):
+class CourseRequirement(BaseModel):
+    course = models.ForeignKey(Course, on_delete=models.CASCADE, related_name='requirements')
+    requirement = models.CharField(max_length=255)
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)